README.md

```diff
@@ -6,14 +6,14 @@
 
 # Ankaios Python SDK for Eclipse Ankaios
 
-Eclipse Ankaios provides workload and container orchestration for automotive
+[Eclipse Ankaios](https://github.com/eclipse-ankaios/ankaios) provides workload and container orchestration for automotive
 High Performance Computers (HPCs). While it can be used for various fields of
 applications, it is developed from scratch for automotive use cases and provides
 a slim yet powerful solution to manage containerized applications.
 
 The Python SDK provides easy access from the container (workload) point-of-view
 to manage the Ankaios system. A workload can use the Python SDK to run other workloads
-and get the state of the Ankaios system. 
+and get the state of the Ankaios system.
```

Comment:
What about: "A workload can use the Python SDK to start, stop and update other workloads and configs and get the state of the Ankaios system."

Reply:
Done.